shared/src/common/enums/permissions/permissions.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum Permissions {
```

Comment:
```suggestion
enum Permission {
```
don't forget about filename